mappings/net/minecraft/nbt/TagCompound.mapping

```diff
@@ -62,7 +62,7 @@ CLASS none/dt net/minecraft/nbt/TagCompound
 	METHOD a setIntArray (Ljava/lang/String;[I)V
 		ARG 0 key
 		ARG 1 value
-	METHOD a copyTo (Lnone/dt;)V
+	METHOD a copyFrom (Lnone/dt;)V
```

Comment:
I was told `addAll` would be a more fitting name to go with `Collection.addAll` or something.
Let me know with a :+1: if you want it changed to that, and I'll update the PR.